src/layout.ts

```diff
@@ -59,7 +65,7 @@ function layoutParagraph(paragraph: Paragraph, box: Box, fonts: Font[]): Frame {
   const maxWidth = box.width - padding.left - padding.right;
   const maxHeight = box.height - padding.top - padding.bottom;
   const innerBox = { x: padding.left, y: padding.top, width: maxWidth, height: maxHeight };
-  const text = paragraph.text && layoutText(paragraph, innerBox, fonts);
+  const text = paragraph.text && layoutText(paragraph, innerBox, fonts, padding);
```

Comment:
Passing the `padding` to `layoutText` should not be needed, as the `innerBox` has already subtracted the padding. To the layout code should only align the content in that inner box.

Reply:
While centering, I want to ignore left and right paddings following the behavior in CSS, however `box.x` only contains padding-left but not the right information.